simplify/src/main/java/org/cf/simplify/Launcher.java

```diff
@@ -179,33 +179,44 @@ private void executeClass(VirtualMachine vm, String className, RuntimeStats stat
                 continue;
             }
 
-            boolean executeAgain;
-            do {
-                System.out.println("(" + stats.getCurrentMethodIndex() + " / " + stats.getTotalMethods() + ") Executing top level method: " + method);
-                ExecutionGraph graph = null;
-                try {
-                    graph = vm.execute(method);
-                } catch (VirtualMachineException e) {
-                    System.err.println("Aborting execution; exception: " + e);
-                }
-
-                if (null == graph) {
-                    System.out.println("Skipping optimization of " + method + "; null execution graph");
-                    stats.incrementFailedMethodCount();
-                    break;
-                }
+            try {
```

Comment:
According to the help, the option should only ignore errors during optimization. This looks like you want to ignore any possible error in execution, optimization, and moving data around. Is that the case? You want to ignore ANY error?

There are two main places an error might happen: vm.execute() and optimizer.simplify(). Would be better to have a try/catch around _just_ those lines and the catch should be specific. E.g. "Unexpected, non-virtual execution executing ..., aborting" and "Exception optimizing ..., skipping"

I like the idea of adding a try/catch layer over everything so simplify will be more robust but I didn't originally add it because it's possible for the VM state to get corrupted in such a way that 1 error will subtly fuck up everything else afterwards.  It's better to skip specific problem methods, but adding this option and making this tradeoff clear in the readme is probably enough.